shared/templates/pam_options/ansible.template

```diff
@@ -24,8 +24,8 @@
 
 - name: Check to see if '{{{ MODULE }}}' module is configured in '{{{ PATH }}}'
   shell: |
-    set -o pipefail
-    grep -E '^\s*{{{ TYPE }}}\s+\S+\s+{{{ MODULE }}}' {{{ PATH }}} | cat
+     set -o pipefail
+     { grep -E '^\s*{{{ TYPE }}}\s+\S+\s+{{{ MODULE }}}' {{{ PATH }}} || true; }
```

Comment:
not sure why this one can't be done as:
shell: grep -E '^\s*{{{ TYPE }}}\s+\S+\s+{{{ MODULE }}}' {{{ PATH }}} || true

But ansible-lint gave a pipe error when I tried that.